Formula/immudb.rb

```diff
@@ -21,8 +21,16 @@ def install
 
   test do
     immudb_port = free_port
+
+    ENV["IMMUDB_AUTH"] = "false"
+    ENV["IMMUCLIENT_AUTH"] = "false"
+
+    tmp_data_dir = "/var/tmp/immudb"
+
+    rm_rf(tmp_data_dir)
```

Comment:
Please just make a directory in the current location, test directories only live temporarily so it'll be automatically removed.

Reply:
thanks @SMillerDev, already changed